site/content/en/docs/Reference/gameserver.md

```diff
@@ -208,7 +208,7 @@ for `address`.)
 The policy for `address` will work for many use-cases, but for some advanced cases, such as IPv6 enablement, you may need
 to evaluate all `addresses` and pick the addresses that best suits your needs.
 
-[addresses]: https://v1-26.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#nodeaddress-v1-core
-[podIPs]: https://v1-26.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#podip-v1-core
+[addresses]: https://v1-28.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#nodeaddress-v1-core
```

Comment:
```suggestion
[addresses]: {{< k8s-api-version href="#nodeaddress-v1-core" >}}
```

Does this not work?

Reply:
No this currently fails: https://pantheon.corp.google.com/cloud-build/builds;region=global/49a4c639-318c-476a-8b60-6ff6811d35bd;step=19?e=13803378&mods=allow_workbench_image_override&project=agones-images

It's complaining something about some colon, but I can't make anything out of it